docs/docs/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Welcome
+---
+
+# Welcome to Invariant
+
+<div class='subtitle'>How to get started with the Invariant SDK</div>
+
+The Invariant SDK enables you to share, analyze and collaborate on AI agent runtime traces. This documentation will guide you through the process of setting up the SDK, capturing traces, and uploading them to the Invariant Explorer.
+
+To get started, first install the SDK by following the instructions in the [Installation](installation.md) section.
```

Comment:
I believe this will be relevant for later once we make the sdk available on pip?